.github/workflows/test-pg_search-benchmark.yml

```
@@ -37,6 +42,19 @@ jobs:
     steps:
       - name: Checkout Git Repository
         uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.commit || 'main' }} # If no commit is provided, use the main branch.
+
+      # We manually fetch the benchmark queries from `main` to make sure all are available when backfilling
+      # old commits potentially created before new queries were added.
+      - name: Fetch the Latest Benchmark Queries
+        run: |
+          git clone https://github.com/paradedb/paradedb.git paradedb-temp
```

Should use a shallow clone here and below:
```suggestion
          git clone --depth 1 --branch main https://github.com/paradedb/paradedb.git paradedb-temp
```